.teamcity/src/main/kotlin/configurations/PartialTrigger.kt

```diff
@@ -36,4 +37,16 @@ class PartialTrigger<T : BaseGradleBuildType>(triggerName: String, triggerId: St
     dependencies {
         snapshotDependencies(dependencies)
     }
+
+    val partialTriggerThis = this
```

Comment:
I think it would be better to add the code here: https://github.com/gradle/gradle/blob/2d01af31255ecdd207f15a307f0260a0aec5b629/.teamcity/src/main/kotlin/projects/StageProject.kt#L100-L102

So it is only for the specific trigger and we don't need to do anything in `PartialTrigger`.

Reply:
Good idea, PTAL.